paper-tabs.html

```diff
@@ -539,6 +540,9 @@
 
       _tabChanged: function(tab, old) {
         if (!tab) {
+          // remove bar without animation
+          this.$.selectionBar.classList.remove('expand');
```

Comment:
Also, we prefer comments to be sentence cased: `// Remove bar without animation.`